diff --git a/src/snippets.ts b/src/snippets.ts
index 6893b315ceb9dcf602e12d8137dd0ac29a57892e..d50f6a97c2acf40431c9a5f787487fd59777e79f 100644
--- a/src/snippets.ts
+++ b/src/snippets.ts
@@ -9,7 +9,7 @@ export type ClientInitOptions = {
   injectImmediately?: boolean;
 } & SpotlightAstroIntegrationOptions;
 
-const buildClientImport = (importPath: string) => `import * as Spotlight from '${importPath}';`;
+const buildClientImport = (importPath: string) => `import * as Spotlight from ${JSON.stringify(importPath)};`;
 
 const buildClientInit = (options: ClientInitOptions) => {
   const integrationCalls = options.integrationNames